spec.bs

```diff
@@ -3948,6 +3956,7 @@ from querying the server during an auction.
 
 <div algorithm>
   To <dfn>update k-anonymity cache for key</dfn> given a [=SHA-256=] |hashCode|:
+    1. [=Assert=] that these steps are running [=in parallel=].

```

Comment:
Wrong CL?

Reply:
No, these are intended changes. The only reason these weren't guaranteed to be in parallel before is because they were called directly from the Abort handler (which doesn't execute in parallel).

Reply:
Ack. (I guess you're the one with the "resolve conversation" button?)